### PR TITLE
Remove and add 'use on file' for address, phone, email

### PR DESCRIPTION
**Pull request by ElleeB**
## Description of change
Veteran address, phone, and email should prompt intake specialist to use the Veteran's address on file

## Original issue(s)
address: https://vajira.max.gov/browse/API-4035
email: https://vajira.max.gov/browse/API-4036
phone: https://vajira.max.gov/browse/API-4037

## Testing
Tests and minimum expected pdf updated

### CODE REVIEW

**Comment by ElleeB** [comment]
Screen Shot 2020-12-17 at 6 05 03 PM

File: modules/appeals_api/spec/workers/notice_of_disagreement_pdf_submit_job_spec.rb [hunk 1]
```diff
@@ -103,7 +102,6 @@
       generated_pdf = described_class.new.generate_pdf(notice_of_disagreement.id)
       reader = PDF::Reader.new(generated_pdf)
       expect(reader.pages.size).to eq 5
-      expect(reader.pages.first.text).to include email
```
Comment (cilestin) [inline modules/appeals_api/spec/workers/notice_of_disagreement_pdf_submit_job_spec.rb:106]: We should make sure the PDF has the "Use `X` on file" content

**Review by cilestin — CHANGES_REQUESTED** [review]
Surprised this didn't require an update to the minimum pdf as well. Does it not have an example email at least that should have changed from `a@a.com` to `use email on file`? 🤔

**Review by cilestin — APPROVED** [review]
LGTM 🎉

**Pull request merged**